graph/graph.py

```diff
@@ -23,13 +24,64 @@ class KytosGraph:
 
     def __init__(self):
         self.graph = nx.Graph()
+        self._accepted_metadata = {
+            'ownership',
+            'bandwidth',
+            'reliability',
+            'priority',
+            'utilization',
+            'delay',
+        }
+        ownership_processor = ProcessEdgeAttribute(
+            'ownership',
+            TypeDifferentiatedProcessor({
+                str: lambda x: frozenset(x.split()),
+                dict: lambda x: frozenset(x.keys()),
+                type(None): None
+            })
+        )
         self._filter_functions = {
-            "ownership": lazy_filter(str, filter_in("ownership")),
-            "bandwidth": lazy_filter((int, float), filter_ge("bandwidth")),
-            "reliability": lazy_filter((int, float), filter_ge("reliability")),
-            "priority": lazy_filter((int, float), filter_le("priority")),
-            "utilization": lazy_filter((int, float), filter_le("utilization")),
-            "delay": lazy_filter((int, float), filter_le("delay")),
+            "ownership": EdgeFilter(
+                operator.contains,
+                UseValIfNone(ownership_processor),
+                TypeCheckPreprocessor(
```

Comment:
All the usages of `TypeCheckPreprocessor` instances except on `ProcessEdgeAttribute`, at this point in time, are just trying to validate an input that has already been validated [at higher levels call sites via the API endpoint](https://github.com/kytos-ng/pathfinder/blob/master/main.py#L85). So, in practice, we shouldn't need to keep revalidating this business logic part here (back then we had to, but not anymore), otherwise it's just more code to maintain and understand. It's great to maintain the new `EdgeFilter` with the exact same structure and ready to also set a callable preprocessor, but until we really need, to need to start passing these here. Less overhead in terms of code, and as you know, code is read way more times than it's written, so let's prioritize for that, while also evolving the way you're envisioning this part. Let me know if you had any other strong reason to still maintain, otherwise let's remove passing `TypeCheckProcessor` to `EdgeFilter` here and in the other attributes that are type checking against the `value`

Reply:
I completely forgot about the API validation.  Yeah, I'll get rid of usage of the `TypeCheckPreprocesor`.